### ISSUE TITLE
Does recognition text score normalise between zero to one?

### ISSUE THREAD

**Issue by balandongiv** [open-mmlab/mmocr#1332]
In mmocr `'0.6.0'` (I maybe wrong about the version since I dont have the old copy in hand due to recently upgrade to 0.6.1 without making a backup of the old mmocr) , I notice the score for every character is in range [0, 1]  This apply for both `SAR` and `RobustScanner`.

However, in  `'0.6.1'` , the score was cap around 0.029 for both the  `SAR` OR `RobustScanner`. Is this a a raw logits score? 

May I know whether there is code changes made from `0.6.0` to `'0.6.1'` that result in this new behavior?

The issue can reproduce via the following [notebook](https://github.com/balandongiv/mmocr_tutorial/blob/main/normal_mmocr.ipynb). Should there is changes made, may I know how to get the score in range [0, 1] 

Thanks in advance for the time taken entertaining this post.

**Comment by xinke-wang** [comment]
Hi, it seems a bug caused by this PR https://github.com/open-mmlab/mmocr/pull/1119 , we'll fix this soon.

**Comment by balandongiv** [comment]
Thanks for the feedback @xinke-wang . While waiting for the fix, may I know what is the earliest version I should revert to avoid this issue?
